blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/PipelineApiTest.java

```diff
@@ -797,13 +799,13 @@ public void parameterizedFreestyleTestWithoutDefaultParam() throws Exception {
                 ), 400);
     }
 
+    @Ignore
```

Comment:
Comment out this test to unblock and manually confirm that this still works.

If you do this, please raise a new ticket to get the test working again.